x-pack/platform/plugins/private/upgrade_assistant/public/application/components/overview/upgrade_step/upgrade_step.tsx

```diff
@@ -145,6 +154,36 @@ const UpgradeStep = () => {
 
       <EuiSpacer size="m" />
 
+      <EuiCallOut title={i18nTexts.securitySolnLogsDBCalloutTitle}>
+        <FormattedMessage
+          id="xpack.upgradeAssistant.overview.securitySolutionLogsDBCalloutBody"
+          defaultMessage="We recommend that Elastic Security users disable logsdb index mode before upgrading ({logsDataStreamDocs}). The logsdb index mode requires extra CPU capacity during the data indexing process. To avoid timeouts and errors for your data ingestion backups and security detection rules, you should disable the logsdb index mode unless you are sure enough additional hot data tier CPU capacity has been provisioned. You can enable it again after the upgrade ({indexModeWithSecuritySoln})."
```

Comment:
```suggestion
          defaultMessage="We recommend that Elastic Security users disable logsdb index mode before upgrading ({logsDataStreamDocs}), if not already using logsdb. The logsdb index mode requires extra CPU capacity during the data indexing process. To avoid timeouts and errors for your security detection rules and/or data ingestion backups, you should disable the logsdb index mode unless you are sure enough additional hot data tier CPU capacity has been provisioned. You can enable it again later after the upgrade ({indexModeWithSecuritySoln})."
```